src/core_pdf/api/compat/unstructured/__init__.py

```
@@ -0,0 +1,647 @@
+        return []
+    length = int(numpy.max(boxes[:, axis::2]))
+    projection = numpy.zeros(max(0, length), dtype=numpy.int64)
+    for box in boxes:
+        projection[int(box[axis]) : int(box[axis + 2])] += 1
+    occupied = numpy.where(projection > 0)[0]
+    if not len(occupied):
+        return []
+    gaps = numpy.where(occupied[1:] - occupied[:-1] > 1)[0]
+    starts = [int(occupied[0]), *(int(occupied[index + 1]) for index in gaps)]
+    ends = [*(int(occupied[index]) + 1 for index in gaps), int(occupied[-1]) + 1]
+    return list(zip(starts, ends))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)